Sources/Core/Sources/Render/Shader/ChunkShaders.metal

```diff
@@ -89,3 +89,38 @@ fragment float4 chunkFragmentShader(RasterizerData in [[stage_in]],
 
   return color;
 }
+
+
+constant float2 quadVertices[] = {
+     float2(-1.0,  1.0),
+     float2(-1.0, -1.0),
+     float2( 1.0, -1.0),
+     float2(-1.0,  1.0),
+     float2( 1.0,  1.0),
+     float2( 1.0, -1.0)
+ };
+
+ struct QuadVertex {
+     float4 position [[position]];
+     float2 uv;
+ };
+
+
+ fragment float4 screenFragmentFunction(QuadVertex v [[stage_in]],
+                                  texture2d <float> outputImage [[ texture(0) ]]) {
```

Comment:
This indentation is a bit funky (at least it is in the GitHub UI, probably depends on your tab width)